maint/local_python/binding_c.py

```diff
@@ -531,6 +531,10 @@ def process_func_parameters(func):
             func['_has_comm'] = name
         elif name == "win":
             func['_has_win'] = name
+        elif name == "session" and p['param_direction'] == 'out':
+            func['_has_session_init'] = name
```

Comment:
`'_has_xxx'` means having `xxx` parameter, so `_has_session_init` doesn't fit. This is just for `MPI_Session_init`, right? I think you can skip here, and directly test `func['name']` later.

Reply:
Thanks for this hint. Changed the code accordingly.